src/common/closest-parent/index.js

```diff
@@ -0,0 +1,32 @@
+// https://developer.mozilla.org/en-US/docs/Web/API/Element/matches
+
+function closestParent(el, selector) {
```

Comment:
Sorry to be totally OCD here, but should this module name be a bit more descriptive, like `closestAncestor` or `closestAncestorMatch` or something like that? It does iterate upwards through ancestors, right?

Reply:
To me "parents" and "ancestors" mean the same thing. 🤷‍♂️  But stating `match` does make sense, as that's what it's doing.

Reply:
There's actually this, too: https://developer.mozilla.org/en-US/docs/Web/API/Element/closest

I think this is better, since it's already part of the JS spec. Thoughts?

Reply:
Parents are direct ancestors, one node up ;-)

Reply:
Closest is better if you can find a polyfill for it.

Reply:
Yeah, working on updating the PR.